templates/base/packages/nextjs/scaffold.config.ts.template.mjs

```diff
@@ -11,7 +11,7 @@ export type ScaffoldConfig = {
 
 const scaffoldConfig = {
   // The network where your DApp lives in
-  targetNetwork: chains.hardhat,
+  targetNetwork: chains.${namedArgs?.default?.[0]?.chainName || "mainnet"},
```

Comment:
Maybe this can improved ? 

Added this condition because people might select `None` as option while choosing template 

Is there a better way ? Can we pass some default value to this templating function ?

Reply:
I've given a general answer for this question and your other comment. I'll mark the two as resolved to avoid having multiple threads, but feel free to continue the conversation in the main thread.